corehq/apps/app_manager/management/commands/benchmark_build_times.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import json
+from datetime import datetime
+
+from django.core.management import BaseCommand
+
+from corehq.apps.app_manager.dbaccessors import get_app
+from corehq.apps.users.models import WebUser
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'domain_app_id_pairs',
+            type=json.loads,
+        )
+        parser.add_argument('username')
+
+    def handle(self, domain_app_id_pairs, username, **options):
+        user_id = WebUser.get_by_username(username).get_id
+        comment = 'Generated via command line for build performance benchmarking.'
+        for (domain, app_id) in domain_app_id_pairs:
+            print("%s: %s" % (domain, app_id))
+            with Timer():
+                app = get_app(domain, app_id)
+                errors = app.validate_app()
+                assert not errors
```

Comment:
Should change to `assert not errors, errors`—seems like it would be very nice to know what errors occurred if this fails.